### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by Big-Hdub**
Added Group details page and added to the store for needed data.  Still working on  the logic for the rendering of certain elements based on conditions.

### CODE REVIEW

File: frontend/src/components/GroupDetailsPage/EventDetails.jsx [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+                        <p className="group-details-event-date">{year}-{month}-{day}</p>
+                        <p className="group-details-event-date centered-dot">.</p>
+                        <p className="group-details-event-date">{time} {amOrPm}</p>
+                    </span>
+                    <h2 className="group-details-event-h2">{name}</h2>
+                    <p className="group-details-event-p">{city}, {state}</p>
+                </div>
+            </div>
+            <div className="group-details-event-bottom">
+                <p className="group-details-event-p">Lorem ipsum dolor sit amet consectetur adipisicing elit. Eaque, perferendis! Nam fugit aliquam voluptatum eius aut nobis laudantium iste. Quam natus accusantium, recusandae deleniti voluptates rem quis assumenda! Voluptatibus, est.
```
Comment (AlanDeleon88) [inline frontend/src/components/GroupDetailsPage/EventDetails.jsx:26]: Are you going to render the same text for each event? Or do you plan to render the data from event.descriptions eventually?

**Review by philling83 — CHANGES_REQUESTED** [review]
For your createSelector, your output function shouldn't simply return the inputs (see caution tip in docs: https://redux.js.org/usage/deriving-data-selectors#createselector-overview). You will want to perform some manipulation of the output. Easiest would be returning the values of the selectedObj

**Review by AlanDeleon88 — COMMENTED** [review]
looks alright to me, double check with phil just in case.

File: frontend/src/components/GroupDetailsPage/EventDetails.jsx (hunk 1 above)
Comment (Big-Hdub) [reply frontend/src/components/GroupDetailsPage/EventDetails.jsx:26]: I might have missed that I have descriptions to display lol.

Comment (Big-Hdub) [reply frontend/src/components/GroupDetailsPage/EventDetails.jsx:26]: Oh I forgot that getting the Event by Id gets some more information.

**Pull request merged**